test/plugin/test_out_elasticsearch.rb

```diff
@@ -789,8 +827,9 @@ def test_uses_no_subsecond_precision_by_default
     rescue
       time = Fluent::Engine.now
     end
-    driver.emit(sample_record, time)
-    driver.run
+    driver.run(default_tag: 'test') do
+      driver.feed(time.to_i, sample_record)
+    end
     assert(index_cmds[1].has_key? '@timestamp')
     assert_equal(index_cmds[1]['@timestamp'], Time.at(time).iso8601)
```

Comment:
Default value of `@time_precision` is 9.
I guess that 

```ruby
assert_equal(index_cmds[1]['@timestamp'], Time.at(time).iso8601(9))
```

is correct assertion.